huggan/pytorch/dcgan/train.py

```diff
@@ -41,8 +41,8 @@
 
 from datasets import load_dataset
 
-from metrics.inception import InceptionV3
-from metrics.fid_score import calculate_fretchet
+from huggan.pytorch.metrics.inception import InceptionV3
```

Comment:
So did you remove the metrics from the dcgan folder, and place them in the Pytorch folder?